clean_project/project/index.py

```diff
@@ -72,6 +79,9 @@ def build_index(index_words, connect_path):
                         else:
                             data_insert = (word, file_path, index_words[word].get(file_path))
                             cursor.execute(QUERY_INSERT, data_insert)
+                        count += 1
+                        if count % 1000 == 0:
+                            print count
```

Comment:
it would be good to use a logging module instead of print, and provide a useful message